### PR TITLE
feat: improve simulator loading and empty states

### PR DESCRIPTION
**Pull request by MeLlamoPablo**
https://www.notion.so/prepo/Simulator-default-value-invalid-state-eeb96010e762433e841071b0af3c9a8b

https://user-images.githubusercontent.com/11708035/236011790-050fb0e1-aa87-4be9-8148-942ebf9d13b7.mp4

### CODE REVIEW

File: apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx [hunk 1]
```diff
@@ -52,6 +55,15 @@ const MarketNotAvailable = styled.span`
   color: ${({ theme }): string => theme.color.neutral2};
 `
 
+const SimulatorSkeleton = styled(Skeleton).attrs({
+  width: '232px',
+  height: '136px',
+})`
```
Comment (chrisling-dev) [inline apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx:61]: skeleton has a weird width on mobile, could we try `width: '100%'`?

```suggestion
const SimulatorSkeleton = styled(Skeleton).attrs({
  width: '100%',
  height: '136px',
})`
```

**Review by chrisling-dev — CHANGES_REQUESTED** [review]
Some minor comments

File: apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx [hunk 2]
```diff
@@ -115,15 +121,18 @@ const TradeSimulator: React.FC = () => {
...
+          <Skeleton width="120px" height="22px" />
+        </Flex>
+      ) : (
+        <ProfitLossValue
+          pnlPercentage={pnlPercentage}
+          tradeSize={
+            action === 'open' ? +(openTradeAmountInWstETH ?? '0') : closePositionValueByCostBasis
+          }
+        />
+      )}
```
[the remark this answers is not in the document]
Comment (MeLlamoPablo) [reply apps/frontend/core-dapp/src/features/simulator/TradeSimulator.tsx:135]: Why not? I think it looks good

File: apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx (hunk 1 above)
Comment (MeLlamoPablo) [reply apps/frontend/core-dapp/src/features/simulator/SimulatorCard.tsx]: Funny, I tried this initially but it didn't work for some reason. It does work now.

**Review by chrisling-dev — APPROVED** [review]
LGTM

**Pull request merged**